src/include/OpenImageIO/simd.h

```diff
@@ -634,6 +634,34 @@ class int4 {
 #endif
     }
 
+    OIIO_FORCEINLINE void load (const int *values, int n) {
+#if defined(OIIO_SIMD_SSE)
+        switch (n) {
+        case 1:
+            m_vec = _mm_castps_si128 (_mm_load_ss ((const float *)values));
```

Comment:
Here's my favorite web site: https://software.intel.com/sites/landingpage/IntrinsicsGuide/

Type "_mm_load_" into the search bar, and you'll see that, strangely, _mm_load_ps takes a float*, whereas the analogous _mm_load_si128 takes a __mi128i* rather than an int*.